grafana-plugin/src/utils/hooks.tsx

```diff
@@ -74,6 +78,56 @@ export function useDebouncedCallback<A extends any[]>(callback: (...args: A) =>
   };
 }
 
+export const useDrawer = <DrawerKey extends string, DrawerData = unknown>(initialDrawerData?: DrawerData) => {
+  const [openedDrawer, setOpenedDrawer] = useState<DrawerKey>(LocationHelper.getQueryParam('openedDrawerKey'));
```

Comment:
a bit biased but maybe `openedDrawerKey` instead?

Reply:
Sure, we can, I'll do it